src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -47,10 +47,12 @@ public Command setAngle(IntakePose intakePose) {
     }
 
     public Command setRollerSpeed(double speed) {
+        inputs.RollerSpeedSetPoint = Units.RotationsPerSecond.of(speed).mutableCopy();
         return runOnce(() -> io.setRollerSpeed(speed));
     }
 
     public Command setCenterRollerSpeed(double speed) {
+        inputs.CenterRollerSpeedSetPoint = Units.RotationsPerSecond.of(speed).mutableCopy();
```

Comment:
I see you enjoy uppercase letters.

Reply:
37e78db06f0f4929c058616a77c98c61ae9877f3